csrc/expr_simplifier.cpp

```diff
@@ -992,16 +991,14 @@ Val* divideFactorized(Val* x, Val* y) {
   auto x_factors = getConstAndSymbolicFactors(x);
   auto y_factors = getConstAndSymbolicFactors(y);
 
-  int64_t quoient_const_factor;
-  std::vector<Val*> quoient_symbolic_factors;
-
   if (*x_factors.first->getInt() % *y_factors.first->getInt() != 0) {
     // not divisible
     return nullptr;
-  } else {
-    quoient_const_factor =
-        *x_factors.first->getInt() / *y_factors.first->getInt();
   }
+  int64_t quoient_const_factor =
```

Comment:
@zasdfgbnm I separated the variable initialization from the short-circuit return.